diff --git a/configure/os/CONFIG_SITE.Common.linux-x86_64 b/configure/os/CONFIG_SITE.Common.linux-x86_64
index 834da1e..2e6f402 100644
--- a/configure/os/CONFIG_SITE.Common.linux-x86_64
+++ b/configure/os/CONFIG_SITE.Common.linux-x86_64
@@ -16,8 +16,7 @@
 
 
 # Use GNU Readline if the header file is installed
-COMMANDLINE_LIBRARY = $(strip $(if $(wildcard \
-    $(GNU_DIR)/include/readline/readline.h), READLINE, EPICS))
+COMMANDLINE_LIBRARY = READLINE
 
 # If libreadline needs additional libraries to be linked with it, try
 # uncommenting each of the lines below in turn, starting with the top
